src/Keboola/ConfigMigrationTool/Service/OrchestratorService.php

```diff
@@ -13,10 +13,10 @@ class OrchestratorService
     /** @var Client */
     private $client;
 
-    public function __construct()
+    public function __construct(?string $baseUrl = 'https://syrup.keboola.com/orchestrator/')
```

Comment:
Ty defaulty bych dal pryč všude kde jsi je nastavil. Nic jako default stack nemáme. Tohle je přesně ta situaci kdy to pak člověk zapomene někde předat a pak to najednou nebude fungovat v jiném stacku. Tímhle jsem při přípravě multi stack prostředí trávil nejvíc času:)

Reply:
Jo, myslel som si to, vyhodim to :)